.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: main
+
+on: push
+
+jobs:
+  test:
+    # Skip this job if the head commit’s message contains the string “[skip ci]”.
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    name: Test
+    runs-on: ubuntu-latest
```

Comment:
Shouldn't we run on the same version that is used to build?

Reply:
ubuntu-latest is currently Ubuntu 20.04 (see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners). I don’t see an issue with this changing independent from the Netlify build images because there is really extremely little that could break due to a build image switch in this project. None of the npm packages we use need to be built on the OS plus the `-latest` image won’t just upgrade with the release of a new Ubuntu version as GitHub curates these images like Netlify does.

So, all in all, it’s just one less manual step that we have to do once GitHub deprecates their image for Ubuntu 20.04.